napari/layers/points/points.py

```diff
@@ -1271,6 +1274,31 @@ def mode(self) -> str:
         Mode.TRANSFORM: 'standard',
     }
 
+    @staticmethod
+    def _mode_help(mode):
```

Comment:
Can you add a docstring to these functions? At first I didn't understand what they were doing — just making the help string, right? I might suggest `_mode_help_string`?

I might also suggest something like the following:

```
"quick press <{shortcut}> to switch to {mode} ({desc}), hold and release <{shortcut}> to temporarily switch while holding"
```

Reply:
> `"quick press <{shortcut}> to switch to {mode} ({desc}), hold and release <{shortcut}> to temporarily switch while holding"`

This will be to long. Such text will occupy the whole bottom for one shortcut bar without a place for information about other shortcuts. 
I see that it may increase discoverability for the new mode, but for me, it should instead help to remind about currently set shortcuts, and information about long hold should be distributed another way.

Reply:
oh I thought this text was used for a tooltip?

Reply:
No. It is used on statusbar. 

![Zrzut ekranu z 2022-07-23 16-11-38](https://user-images.githubusercontent.com/3826210/180608763-266cf30c-111e-4955-aa77-6427f470ea93.png)

Here docstring from base `Layer` (but not copied to inheriting class)

```
    help : str
        Displayed in status bar bottom right.
```

Reply:
oh yeah that's crazy, and they all get concatenated together. Maybe we should rethink that anyway... But ok how it is for now. 🙏

Reply:
Should I open a separate issue to not lose this information?